azure-pipelines.yml

```diff
@@ -7,12 +7,19 @@ pr:
   - rel/*
 
 pool:
-  vmImage: vs2017-win2016
+  vmImage: windows-2022
 
 variables: 
   BuildConfiguration: Release
-
+  
 steps:
+- task: UseDotNet@2
+  displayName: 'Use .NET 6 sdk'
+  inputs:
+    packageType: sdk
+    version: 6.0.101
+    installationPath: $(Agent.ToolsDirectory)/dotnet
```

Comment:
Why do we need to specify this?

Reply:
i don' t think its necessary actually.  i added it while attempting to figure out why the CI build was failing (which turned out to be related to the outdated Nuget packages, and not the pipeline itself).